### PR TITLE
Don't use initialConfig when checking immutableProps

### LINKED ISSUE #3048: Cannot read property 'immutableProps' of undefined
I don't know why i'm getting this error. Someone could help how to track the problem?

Cannot read property 'immutableProps' of undefined

Thanks!

### PR DESCRIPTION
**Pull request by gustavohenke**
This parameter may be undefined, as all configs can be defined via props.

Fixes #3048

### CODE REVIEW

**Comment by gustavohenke** [comment]
The Travis error seems to be the same as https://github.com/erikras/redux-form/pull/3051#issuecomment-307969311 🤔

**Comment by erikras** [comment]
```
rm -rf node_modules package-lock.json
npm install
```
on `master` fixed the build on these PRs. `<shrug/>`

File: src/__tests__/reduxForm.spec.js [hunk 1]
```diff
@@ -100,6 +100,37 @@ const describeReduxForm = (name, structure, combineReducers, expect) => {
...
+      }
+
+      const dom = TestUtils.renderIntoDocument(<Container />)
+
+      expect(() => {
+        const button = TestUtils.findRenderedDOMComponentWithTag(dom, 'button')
+        TestUtils.Simulate.click(button)
+      }).toNotThrow()
+    })
+
```
Comment (naw) [inline src/__tests__/reduxForm.spec.js]: Nice! Thanks for adding a spec for this. The original change that broke this certainly seemed innocent, so this spec will help prevent similar regressions in the future.

**Comment by MichaelDeBoey** [comment]
@erikras Any idea when this will be released? 🙂

**Comment by erikras** [comment]
Published in [`v7.0.0`](https://github.com/erikras/redux-form/releases/tag/v7.0.0).

**Pull request merged**